llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -17741,7 +17749,7 @@ static MachineBasicBlock *emitSelectPseudo(MachineInstr &MI,
   auto InsertionPoint = TailMBB->begin();
   while (SelectMBBI != SelectEnd) {
     auto Next = std::next(SelectMBBI);
-    if (isSelectPseudo(*SelectMBBI)) {
+    if (isSelectPseudo(*SelectMBBI) || MI.getOperand(2).isImm()) {
```

Comment:
Can you explain this change? There are no other references to MI in this loop so I'm not sure what MI's operand has to do with anything.

Reply:
try to generate the phi node. RISCV::Select_GPR_Using_CC_Imm is added to the isSelectPseudo function now.